chengelog.txt

```diff
@@ -1,9 +1,6 @@
-1. ������� ����� �� ������������. ���������� ����������� ��� ���� �����. ������� �������� ����� ���������, �������
```

Comment:
You shouldn't remove the description of the previous task from changelog.txt but should to add new task to the beginning of the file.